### PR TITLE
Update create-compliance-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
update the statement to reflect that you cannot enroll a single device to multiple users at the same time.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1706

**Comment by PRMerger9** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/protect/create-compliance-policy.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ To use device compliance policies, be sure you:
 
 - Enroll devices in Intune (required to see the compliance status)
 
-- Enroll devices to one user, or enroll without a primary user. Devices enrolled to multiple users aren't supported.
+- Enroll devices to one user, or enroll without a primary user. Single Device cannot be enrolled  multiple users.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/create-compliance-policy.md:64]:
```suggestion
- Enroll devices to one user, or enroll without a primary user. Single devices cannot be enrolled to multiple users.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger9** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Brenduns** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**